apps/poker-roll/src/main.tsx

```diff
@@ -1,13 +1,41 @@
 import { StrictMode } from 'react';
 import * as ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
+
+// import Cashgame 
+// import Bankroll 
+// import Tournament 
```

Comment:
Need to specify where these are being imported from like your other imports here.

`import Cashgame from './path/to/component'`

Reply:
still isn't liking the pathing ive put in, but i have fleshed them out to be what i expect them to be

Reply:
Ah gotcha, looks like a TypeScript issue. I'm going to push a config tweak really quick.